config/routes.rb

```diff
@@ -21,6 +21,9 @@
         end
       end
     end
+    namespace :v1 do
+      resources :stories, only: [:create, :update, :index, :show, :destroy]
```

Comment:
I think we definitely want to do a soft destroy, to either:
- Exclude the stories that are "deleted" from showing up for the customer 
- Mark the story for deletion and go through our own back office processing for deleting stories

Either way, we want to include in our terms that the data is at our discretion, with the exception of customers making a request to have their data deleted, which then we will comply with within 30 days or something like that

Reply:
Agreed. Destroy currently is not implemented in the controller. I don't think should be mvp

Reply:
The data is kind of our only asset to start with (per our chat from earlier)
- To train our own LLM
- To monetize for publication

Reply:
Definitely. I envision an `status` column maybe managed by a state machine